### DISCUSSION TITLE
Does it works with podman?

### DISCUSSION THREAD

**Discussion by nekogochan** [ajayd-san/gomanagedocker#32] [Q&A] [answered]
_Most users can simply alias Docker to Podman (alias docker=podman) without any problems_

https://docs.podman.io/en/latest/

**Comment by ajayd-san** [comment, marked as answer]
Hey! 

Unfortunately no 😔.

I use the underlying docker API to interact with the daemon so aliasing doesn't do anything. However, this would be a cool thing to add in the future. Can you create an issue? I will try to add it soon!
